JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.ts

```diff
@@ -0,0 +1,278 @@
+        this.colorPicker = component;
+      },
+    };
+
+    this.fontSizeOptions = {
+      displayExpr: 'text',
+      keyExpr: 'size',
+      useSelectMode: true,
+      items: this.fontSizes,
+      selectedItemKey: this.fontSizes[2].size,
+      itemTemplate: 'fontSizeTemplate',
+    };
+
+    this.lineHeightOptions = {
+      icon: 'indent',
+      displayExpr: 'text',
+      keyExpr: 'lineHeight',
+      useSelectMode: true,
+      items: this.lineHeights,
+      selectedItemKey: this.lineHeight,
+      onSelectionChanged: (e: { item: { lineHeight: number } }): void => {
+        this.lineHeight = [e.item.lineHeight];
+      },
+    };
+
+    this.fontSelectorOptions = {
+      placeholder: 'Font',
+      displayExpr: 'text',
+      dataSource: new DataSource(this.fonts),
+    };
+
+    this.fontStyleOptions = {
+      displayExpr: 'text',
+      items: this.fontStyles,
+      keyExpr: 'style',
+      stylingMode: 'outlined',
+      selectionMode: 'multiple',
+      onItemClick: (e: { itemData: { hint: string } }): void => {
+        notify(`The "${e.itemData.hint}" button was clicked`);
+      },
+    };
+
+    this.listOptions = {
+      items: this.listTypes,
+      keyExpr: 'alignment',
+      stylingMode: 'outlined',
+      onItemClick: (e: { itemData: { hint: string } }): void => {
+        notify(`The "${e.itemData.hint}" button was clicked`);
+      },
+    };
+
+    this.dateBoxOptions = {
+      width: 200,
+      type: 'date',
+      value: new Date(2022, 9, 7),
+    };
+
+    this.checkBoxOptions = {
+      value: false,
+      text: 'Private',
+      onOptionChanged: (): void => {
+        notify('Private option has been changed!');
+      },
+    };
+
+    this.attachOptions = {
+      icon: 'attach',
+      text: 'Attach',
+      onClick: (): void => {
+        notify('Attach button has been clicked!');
+      },
+    };
+
+    this.addButtonOptions = {
+      icon: 'add',
+      text: 'Add',
+      onClick: (): void => {
+        notify('Add button has been clicked!');
+      },
+    };
+
+    this.removeButtonOptions = {
+      icon: 'trash',
+      text: 'Remove',
+      onClick: (): void => {
+        notify('Remove button has been clicked!');
+      },
+    };
+
+    this.aboutButtonOptions = {
+      icon: 'help',
+      text: 'About',
+      onClick: (): void => {
+        notify('About button has been clicked!');
+      },
+    };
+
+    this.textAlignOptions = {
+      keyExpr: 'alignment',
+      stylingMode: 'outlined',
+      items: this.getTextAlignItems(true),
+      selectedItemKeys: this.selectedTextAlign,
+      onItemClick: (e: { itemData: { hint: string } }): void => {
+        notify(`The "${e.itemData.hint}" button was clicked`);
+      },
+    };
+  }
+
+  onColorClick(color: any) {
+    this.colorPicker
+      .element()
+      .getElementsByClassName('dx-icon-square')[0].style.color = color;
+    this.colorPicker.close();
+  }
+
+  onChangeHandler({ value }) {
```

Comment:
This name does not describe when the handler is called.
```suggestion
  onToolbarLineModeChange({ value }) {
```

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/5ba43954aa5e53c1a0c2a8596281ef2bf941d78c